examples/sig_test.go

```diff
@@ -13,8 +13,8 @@ import (
 
 type Suite interface {
 	kyber.Group
-	Cipher(key []byte, options ...interface{}) kyber.Cipher
 	kyber.Encoding
+	kyber.XOFFactory
```

Comment:
I remember you were not fond of the `Factory`s - any better suggestion?

Reply:
Nope, it does what it says, so no complaint from me. Based on Java nightmares, I don't like factories, but kyber uses them in a reasonable way so let's call it what it is.